plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/schema/confluent/TestKafkaWithConfluentSchemaRegistryMinimalFunctionality.java

```diff
@@ -114,6 +115,33 @@ public void testTopicWithKeySubject()
                         .buildOrThrow());
     }
 
+    @Test
+    public void testTopicWithTombstone()
+    {
+        String topicName = "topic-tombstone-" + randomNameSuffix();
+
+        assertNotExists(topicName);
+
+        Map<String, String> producerConfig = schemaRegistryAwareProducer(testingKafka)
+                .put(KEY_SERIALIZER_CLASS_CONFIG, KafkaAvroSerializer.class.getName())
+                .put(VALUE_SERIALIZER_CLASS_CONFIG, KafkaAvroSerializer.class.getName())
+                .buildOrThrow();
+
+        List<ProducerRecord<Long, GenericRecord>> messages = createMessages(topicName, 2, true);
+        testingKafka.sendMessages(messages.stream(), producerConfig);
+
+        // sending tombstone message (null value) for existing key
+        testingKafka.sendMessages(LongStream.of(1).mapToObj(id -> new ProducerRecord<>(topicName, id, null)), producerConfig);
+
+        waitUntilTableExists(topicName);
+
+        // tombstone message should have all null values except key
```

Comment:
do you think a test which verifies that tombstones can be differentiated from messages with all fields set to `null` would be useful? It would help us "document" the contract which we talked about so that users can differentiate between tombstones and actual nulls in their messages.

Reply:
Test added, we can differentiate these 2 cases by internal message_corrupt field